config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/keystore/KeystoreHelper.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.util.keystore;
+
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.PrivateKey;
+import org.bouncycastle.asn1.pkcs.PrivateKeyInfo;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.bouncycastle.openssl.PEMParser;
+import org.bouncycastle.openssl.jcajce.JcaPEMKeyConverter;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.io.*;
+import java.nio.file.Files;
+import java.security.*;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.util.Collection;
+
+import static com.ca.apim.gateway.cagatewayconfig.util.gateway.CertificateUtils.loadCertificateFromFile;
+import static java.security.KeyStore.getInstance;
+import static java.util.Arrays.sort;
+import static java.util.Arrays.stream;
+import static java.util.Comparator.comparing;
+import static java.util.Optional.ofNullable;
+
+@Singleton
+public class KeystoreHelper {
+
+    // TODO add a proper password
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=76&issues=AWZZr2vi9tx2us2yAaRm&open=AWZZr2vi9tx2us2yAaRm)

[ISSUE_KEY]: AWZZr2vi9tx2us2yAaRm